concrete/src/File/Importer.php

```diff
@@ -148,6 +150,14 @@ public function __construct()
             $processor->setRescanThumbnails(false);
             $this->addImportProcessor($processor);
         }
+        if ($config->get('concrete.file_manager.images.svg_sanification.enabled')) {
+            $processor = $this->app->build(SvgSanitizerProcessor::class);
```

Comment:
Why `build` and not `make`? Using `build` makes it really hard to override this implementation